build.sbt

```diff
@@ -13,30 +13,33 @@ ThisBuild / bintrayRepository := "maven"
 
 ThisBuild / licenses += ("MIT", url("https://opensource.org/licenses/MIT"))
 
+// don't publish the jars for the root project (http://stackoverflow.com/a/8789341)
+publish / skip := true
+publishLocal / skip := true
+
 /**
   * Semantic versioning attempts to validate that the version generated makes sense relative to previous
   * versions released. We are introducing support for new Scala versions in this release, so the semVerCheck
   * will fail. This setting will ensure that we don't forget to re-enable it after this release.
   */
 val suppressSemVerCheckOfNewScalaVersionsUntilNextVersion = semVerCheck := {
   version.value match {
-    case VersionNumber(Seq(1, 1, x, _*), _, _) if x <= 3 => Def.task {}
+    case VersionNumber(Seq(1, 1, 3 | 4, _*), _, _) => Def.task {}
+    case VersionNumber(Seq(1, 2, 0, _*), _, _) => Def.task {}
```

Comment:
if you're next releasing 1.1.4, why suppress semver for scala 2_12 for 1.2.0 as well?

Reply:
I plan to release it as 1.2.0. Not sure why I added both.

Reply:
Not sure whether I should release as 1.1.4 or 1.2.0